### ISSUE TITLE
Error when running `yarn install` but only on Windows

### ISSUE THREAD

**Issue by felipellrocha** [apollographql/apollo-client#10770]
Installing works on both linux and mac. On windows I get this error:
```
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning " > @monaco-editor/react@4.5.0" has unmet peer dependency "monaco-editor@>= 0.25.0 < 1".
warning "@monaco-editor/react > @monaco-editor/loader@1.3.3" has unmet peer dependency "monaco-editor@>= 0.21.0 < 1".
warning " > @graphql-codegen/typescript-react-apollo@3.3.7" has unmet peer dependency "graphql-tag@^2.0.0".
warning " > @storybook/react@6.5.16" has unmet peer dependency "require-from-string@^2.0.2".
warning " > @testing-library/react@13.4.0" has incorrect peer dependency "react@^18.0.0".
warning " > @testing-library/react@13.4.0" has incorrect peer dependency "react-dom@^18.0.0".
warning " > react-test-renderer@18.2.0" has incorrect peer dependency "react@^18.2.0".
[4/4] Building fresh packages...
$ ts-node .erb/scripts/check-native-dep.js && electron-builder install-app-deps && cross-env NODE_PATH=src NODE_ENV=development TS_NODE_TRANSPILE_ONLY=true webpack --config ./.erb/configs/webpack.config.renderer.dev.dll.ts
  • electron-builder  version=23.6.0
  • loaded configuration  file=package.json ("build" field)
  • installing production dependencies  platform=win32 arch=x64 appDir=C:\buildkite-agent\builds\buildserver-windows-1-2\ultimate-games\build\studio\release\app
Node env: development
[]
ERROR in ./node_modules/graphql/language/printer.js 8:19-46
Module not found: Error: Can't resolve './blockString.js' in 'C:\buildkite-agent\builds\buildserver-windows-1-2\ultimate-games\build\studio\node_modules\graphql\language'
 @ ./node_modules/subscriptions-transport-ws/dist/client.js 66:16-51
 @ dll renderer renderer[35]
 
ERROR in ./node_modules/graphql/language/visitor.js 12:17-51
Module not found: Error: Can't resolve '../jsutils/devAssert.js' in 'C:\buildkite-agent\builds\buildserver-windows-1-2\ultimate-games\build\studio\node_modules\graphql\language'
 @ ./node_modules/graphql/language/printer.js 12:15-38
 @ ./node_modules/subscriptions-transport-ws/dist/client.js 66:16-51
 @ dll renderer renderer[35]
 ```

**Comment by jerelmiller** [comment]
Hey @felipellrocha 👋 

I'm not sure I'm seeing any `@apollo/client` related modules in that stack trace. Just to confirm, are you sure you're opening this issue up in the right repo? Looks like the top-most item in the stack trace is from the `graphql` module:

```
ERROR in ./node_modules/graphql/language/printer.js 8:19-46

...

ERROR in ./node_modules/graphql/language/visitor.js 12:17-51
```

**Comment by felipellrocha** [comment]
Heh, sorry. I had copied the wrong set of lines, but `@apollo/client` was definitely involved (and was showing more errors). I solved the issue yesterday. I believe I hit a bad `yarn` cache. Both `graphql` and `@apollo/client` were not delivering `js` files, creating the error`. Clearing the cache solved the issue, and I believe now that this issue was purely `yarn` related. Thanks for the comment, though! :)
